docs/_static/css/custom.css

```diff
@@ -0,0 +1,258 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
```

Comment:
can we move this to our fork of the theme? 

https://github.com/RasaHQ/sphinx_rtd_theme

@tmbo

Reply:
sure :+1:

Reply:
@psharmz gave you write permission to https://github.com/RasaHQ/sphinx_rtd_theme